lib/matplotlib/backend_bases.py

```diff
@@ -3362,3 +3250,129 @@ def set_message(self, s):
             Message text
         """
         pass
+
```

Comment:
Is this block just a big copy/paste? (so we don't have to bother reviewing the code if it is)

Reply:
It is, *except* for the fact that the FigureManager attribute now defaults to FigureManagerBase.